extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -69,6 +69,23 @@ export let InteractiveComponentDef;
  */
 export let InteractiveReactData;
 
+/**
+ * @typedef {{
+ *   brand-favicon: ?string,
+ *   brand-label: string,
+ *   cta-text: ?number,
+ *   product-details: string,
+ *   product-images: string,
```

Comment:
If it's plural, does it need to be an array?

Reply:
+1 
This needs to be an array of strings.